CHANGELOG.md

```diff
@@ -17,6 +17,16 @@
     cart. You can set it through the new `Spree::Config.variant_pricer_class` setting. This
     class also knows which `PricingOptions` class it cooperates with.
 
+    #### Order Merging on Sign In:
+
+    Previously a before filter added in `core/lib/spree/core/controller_helpers/order.rb`
+    would cause sql queries to be used on almost every request in the frontend.
+    If you do not use Solidus Auth you will need to hook into this helper and
+    call `set_current_order` where your user signs in. This merges incomplete
+    orders a user has going with their current cart.  If you do use
+    Solidus Auth you will need to make sure you use a current enough
+    version that includes this explicit call.
```

Comment:
No, that is where open orders, I believe, are associated with the signing in user in order to keep them from being lost.  Orders will now be merged [here](https://github.com/solidusio/solidus_auth_devise/pull/51/files) in solidus auth devise.  Something similar will need to be implemented in other authentication systems.